hassio/addon-view/hassio-addon-info.html

```diff
@@ -247,13 +240,19 @@
   }
 
   openChangelog() {
+    let data;
     this.hass.callApi('get', `hassio/addons/${this.addonSlug}/changelog`)
-      .then((data) => {
-        this.changelog = data;
-      }, () => {
-        this.changelog = 'Error getting changelog';
+      .then((resp) => {
+        data = resp;
```

Comment:
Just return a value and it will be passed as input to the next promise.

```js
// logs 2
Promise.resolve(1).then(result => result + 1).then(console.log)
```